Livecheckables/gtksourceviewmm.rb

```diff
@@ -0,0 +1,4 @@
+class Gtksourceviewmm
+  livecheck :url   => "https://download.gnome.org/sources/gtksourceviewmm/",
+            :regex => /gtksourceviewmm-(2\.([0-8]\d*?)?[02468](?:\.\d+)+)\.t/
```

Comment:
```suggestion
            :regex => /gtksourceviewmm-(2\.([0-8]\d*?)?[02468](?:\.\d+)*?)\.t/
```

Outside of the necessary restriction of the major version, this diverges from the `download.gnome.org` regex just a little bit, so this suggestion brings it back in line.